source/includes/_examples.md.erb

```diff
@@ -70,6 +70,52 @@ include            | candidate              | Since we're fetching latest job ap
 filter[stage-type] | hired                  | Just list job applications marked as hired
 sort               | <span class="no-wrap">-changed_stage_at</span>      | Sort applications by most recent added to current (hired) stage
 
+## Create candidate with a To-Many location relationship
+
+Create a <code>candidates</code> resource with with a <code>To-Many</code> relationship to a `locations` resource set.
+
+<aside class="warning">
+The <code>location</code> resource has to be created before this request is executed.
+</aside>
+
+```http
+POST https://api.teamtailor.com/v1/candidates HTTP/1.1
+Authorization: Token token=abc123abc123
+X-Api-Version: <%= config[:api_version] %>
+
+{"data":{"type":"candidates","attributes":{"email":"jonas@example.com","pitch":"Lorem ipsum dolor sit amet, consectetur adipiscing elit.","last-name":"Jonas","first-name":"Brusman","merge":true},"relationships":{"locations":{"data":[{"id":"1","type":"locations"}]}}}}
+```
+
+```shell
+     curl -X "POST" "https://api.teamtailor.com/v1/candidates" \
+          -H "Content-Type: application/vnd.api+json" \
+          -H "X-Api-Version: <%= config[:api_version] %>" \
+          -H "Authorization: Token token=abc123abc123" \
+          -d $'{
+       "data": {
+         "type": "candidates",
+         "attributes": {
+           "email": "jonas@example.com",
+           "pitch": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+           "last-name": "Jonas",
+           "first-name": "Brusman",
```

Comment:
lol 😄

Reply:
You're famous: https://github.com/Teamtailor/docs/blob/master/source/includes/candidates/_create.md.erb#L22 😄